src/app/components/Slideout/SlideoutButton.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <!-- Centers images -->
+  <button
+    class='border-0'
+     @click="$emit('input')"
+     :aria-label="`Open the ${name} tab of the slideout`"
+  >
+    <center
+    class='border-0 font-weight-bold text-white d-inline-block'
+    @click="$emit('input')"
+  >
+    <img
+    :alt="`${this.selected ? 'Selected' : 'Unselected'} ${name} tab image`"
```

Comment:
```suggestion
    :alt="`${this.selected ? 'Selected' : 'Unselected'} ${name} tab`"
```
Screenreaders already announce "image", best practice is to not use "image"/"icon"/"picture"/etc in alt attributes